### PR TITLE
Andrea campanella net store

### PR DESCRIPTION
**Pull request by ray-milkey**

### CODE REVIEW

File: pkg/northbound/gnmi/gnmi_test.go [hunk 1]
```diff
@@ -48,15 +48,22 @@ func setUp(t *testing.T) (*Server, *manager.Manager, *MockStore) {
...
 	}
 	ctrl := gomock.NewController(t)
-	mockStore := NewMockStore(ctrl)
-	mgr.DeviceStore = mockStore
+	//TODO create mock here.
+	// mockNetworkChangesStore :=
+	// mockDeviceChangesStore :=
+	//TODO assign here
+	//mgr.NetworkChangesStore := mockNetworkChangesStore
+	//mgr.ChangeStore := mockDeviceChangesStore
```
[the remark this answers is not in the document]
Comment (ray-milkey) [reply pkg/northbound/gnmi/gnmi_test.go]: I think we should defer this to a later PR. This requires breaking the mocks out of the manager package, and I would prefer to do that separately.

File: pkg/manager/manager.go [hunk 2]
```diff
@@ -115,6 +124,8 @@ func LoadManager(configStoreFile string, changeStoreFile string, networkStoreFil
 	}
 	log.Info("Configuration store loaded from ", configStoreFile)
 
+	log.Info("Device Changes store connected")
```
[the remark this answers is not in the document]
Comment (ray-milkey) [reply pkg/manager/manager.go]: Will do. I think Git Hub will make yet another PR for that though. Hopefully we can get rid of all of these once the base PR has all the changes.

[the remark this answers is not in the document]
Comment (Andrea-Campanella) [reply pkg/manager/manager.go]: hope so. let me know when you are done, I'll rebase on top of my base one.

**Comment by Andrea-Campanella** [comment]
superseded by #756

**Pull request closed without merging**